sci-libs/libmed/libmed-4.0.0-r1.ebuild

```diff
@@ -22,15 +24,15 @@ SRC_URI="http://files.salome-platform.org/Salome/other/${MY_P}.tar.gz"
 LICENSE="GPL-3 LGPL-3"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="doc fortran mpi python static-libs test"
+IUSE="doc fortran mpi python test"
 
 REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
 RDEPEND="
-	sci-libs/hdf5[fortran=,mpi=]
+	>=sci-libs/hdf5-1.10.0[fortran=,mpi=]
 	mpi? ( virtual/mpi[fortran=] )
 	python? ( ${PYTHON_DEPS} )
 "
-DEPEND="${RDEPEND}
+BDEPEND="${RDEPEND}
```

Comment:
`BDEPEND` does inherit neither `RDEPEND` nor `DEPEND`

Reply:
Ah, I wanted to ask this, when you told me, this needs to be in BDEPEND... but then I was myself pretty sure that it will inherit it too... Bad guess :)